### ISSUE TITLE
Ability to override typescript compilerOptions / broken interpret support

### ISSUE THREAD

**Issue by unlight** [developit/microbundle#309]
To compiler requires set `module` to es2015 or esnext, which is not compatible with [interpret](https://github.com/gulpjs/interpret) (e.g. gulpfile.ts -> ts-node). They mostly requires `module = commonjs`

Need ability to override typescript compilerOptions.

**Comment by unlight** [comment]
Just found in the source
```
tsconfigOverride: {
	compilerOptions: {
		target: 'esnext',
	},
},
```
Looks like you mixed `target` and `module`

**Comment by developit** [comment]
This was broken in 0.7.0 and there is a fix pending in #311

**Comment by ForsakenHarmony** [comment]
see #300
